Improve Account Switching: Support doc updates (#1482)
dnsimple/dnsimple-support: content/articles/guide-dashboard.md

Your dashboard and Domain List let you manage all of your domains across your accounts.

<info>

> You can access your DNSimple dashboard by clicking on the <label>DNSimple</label> logo in the top left corner of the screen.

Reviewer: ```suggestion
You can access your DNSimple dashboard by clicking the <label>DNSimple</label> logo at the top-left corner of the screen.
```
